.gitignore

```diff
@@ -10,4 +10,5 @@
 ###> friendsofphp/php-cs-fixer ###
 /.php_cs
 /.php_cs.cache
```

Comment:
should we remove these gitignore files? i think they are about php-cs-fixer 2 only, or not?

Reply:
You are right, does not make sense to keep them - removed via a68a1b0dc8ea227bcab8a69532f73f088bb40a6b